xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPIGL.cpp

```diff
@@ -77,7 +77,9 @@ bool CRendererVAAPI::Configure(const VideoPicture &picture, float fps, unsigned
 
   for (auto &tex : m_vaapiTextures)
   {
-    tex.Init(interop);
+    // FIXME How do we know which texture type to create without running the interop test every time?
```

Comment:
yes, we need to run the test. a user may have switched the app from one display to another.

Reply:
So what's the point of having the interop test run at app start at all? Shouldn't we register the vaapi renderer always then?

Reply:
> So what's the point of having the interop test run at app start at all?

test if vaapi decoder can handle 10bit. X11 registers it every time a new context gets created.

Reply:
Ah, if X11 does it every time then it makes sense. I'll try to get it sorted out.